src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -844,15 +833,71 @@ private File directoryForContainer(final URI uri) {
      * Verify the bag we are going to import
      *
      * @param bagDir root directory of the bag
-     * @return true if valid
+     * @return the {@link Bag} if valid
      */
-    public static boolean verifyBag(final File bagDir) {
+    public Bag verifyBag(final Path bagDir) {
         try {
-            final Bag bag = BagReader.read(bagDir);
-            BagVerifier.isValid(bag, true);
-            return true;
+            final BagReader bagReader = new BagReader();
+            final Bag bag = bagReader.read(bagDir);
+            // BagVerifier.isValid(bag, true);
```

Comment:
Is there a reason for removing the `BagVerifier.isValid()` call?

Reply:
Initially this branch being done on top of [FCREPO-3184](https://jira.lyrasis.org/browse/FCREPO-3184) which was handling Bag validation but I broke it off on its own. I ended up  using the `verifyBag` method to read the Bag anticipating that the other work will do the full verification. This PR does pull in an updated version of the `bagit-java` library so it should be easy enough to add the `BagVerifier` back in here.

Reply:
@awoods One thing to note about adding in the `BagVerifier` is that we'll need to update some of the unit/integration tests. This is from the bag under `src/test/resources/sample/bag` having an invalid manifest (which had been done intentionally). This can be remedied, of course, but I wanted to mention it as it will have an impact on the diff.